README.md

```diff
@@ -2,7 +2,7 @@
 ---
 <p align="center">
   <strong>
-    <a href="https://opentelemetry-python.readthedocs.io/en/stable/getting-started.html">Getting Started<a/>
```

Comment:
Both of these links don't work for me. I believe the getting started docs exist on [here](https://opentelemetry.io/docs/instrumentation/python/getting-started/) now.